packages/react-components/react-motion/library/src/factories/createMotionComponent.ts

```diff
@@ -67,6 +68,10 @@ export function createMotionComponent<MotionParams extends Record<string, Motion
     const handleRef = useMotionImperativeRef(imperativeRef);
     const elementRef = React.useRef<HTMLElement>();
     const paramsRef = React.useRef<MotionParams>(params);
+    const skipMotions = useMotionBehaviourContext() === 'skip';
+    const optionsRef = React.useRef<{ skipMotions: boolean }>({
+      skipMotions,
+    });
```

Comment:
Please merge `params` into `optionsRef` and ensure that `optionsRef` is updated in an effect